src/parsec_service/stability.md

```diff
@@ -0,0 +1,68 @@
+# Parsec Stability
+
+Work in progress, see the [related issue](https://github.com/parallaxsecond/parsec-book/issues/83).
+
+## Why is stability important?
+
+Parsec is a long-running service that will be configured, deployed, upgraded and maintained over a
+long period of time. The environment will also have client applications that are calling it, and
+persistent state that is collected on storage. Components will be upgraded over time. For Parsec to
+be supportable in enterprise system, it must not be brittle against upgrades. If Parsec is upgraded
+and restarted, client applications should continue working, and all persistent state from prior
+versions should remain valid and usable.
+
+## What does stability mean for the Parsec service?
+
+The following definition of stability is chosen for the Parsec service: *whatever two versions A and
+B of the Parsec binary, B being newer than A, A and B are stable if A can be removed and replaced by
+B without breaking anything*.
+
+Note that we are not looking at stability in the reverse direction: B can not be replaced by A
```

Comment:
That's true! I guess I can put this as an exception on this note that although rollbacks are not considered, new clients should still be able to communicate with old versions of the Parsec service. So that this document considers only service upgrades for all communication channels except for (1) where both upgrades and downgrades are stable.